MAINTAINERS.md

```diff
@@ -199,38 +199,49 @@ This [repository](https://github.com/powsybl/powsybl-metrix) also provides a C++
 This [repository](https://github.com/powsybl/powsybl-case-server) provides a web service for network cases management.
 The [powsybl-case-datasource](https://github.com/powsybl/powsybl-case-datasource) repository is the client part of this service.
 
-**Reviewers:** [geofjamg](https://github.com/geofjamg), [jonenst](https://github.com/jonenst), [antoinebhs](https://github.com/antoinebhs)  
-**Committers:** [geofjamg](https://github.com/geofjamg), [jonenst](https://github.com/jonenst), [antoinebhs](https://github.com/antoinebhs)  
+**Reviewers:** [etiennehomer](https://github.com/etiennehomer), [EtienneLt](https://github.com/EtienneLt)
+**Committers:** [etiennehomer](https://github.com/etiennehomer), [EtienneLt](https://github.com/EtienneLt)
 
 ### powsybl-network-store-server
 This [repository](https://github.com/powsybl/powsybl-network-store-server) provides a persistent implementation of the IIDM grid model in [PostgreSQL](https://www.postgresql.org/) database. The grid model is exposed as a web service.
-The [powsybl-network-store](https://github.com/powsybl/powsybl-network-store) repository is the client part of this service.
 
-**Reviewers:** [geofjamg](https://github.com/geofjamg), [jonenst](https://github.com/jonenst), [antoinebhs](https://github.com/antoinebhs)  
-**Committers:** [geofjamg](https://github.com/geofjamg), [jonenst](https://github.com/jonenst), [antoinebhs](https://github.com/antoinebhs)
+**Reviewers:** [jonenst](https://github.com/jonenst), [antoinebhs](https://github.com/antoinebhs), [EtienneLt](https://github.com/EtienneLt), [SlimaneAmar](https://github.com/SlimaneAmar), [FranckLecuyer](https://github.com/FranckLecuyer)  
+**Committers:** [jonenst](https://github.com/jonenst), [antoinebhs](https://github.com/antoinebhs), [SlimaneAmar](https://github.com/SlimaneAmar), [FranckLecuyer](https://github.com/FranckLecuyer)
+
+### powsybl-network-store
+This [repository](https://github.com/powsybl/powsybl-network-store-server) provides a client for the persistent implementation of the IIDM grid model.
+
+**Reviewers:** [jonenst](https://github.com/jonenst), [antoinebhs](https://github.com/antoinebhs), [EtienneLt](https://github.com/EtienneLt), [SlimaneAmar](https://github.com/SlimaneAmar), [FranckLecuyer](https://github.com/FranckLecuyer)
+**Committers:** [jonenst](https://github.com/jonenst), [antoinebhs](https://github.com/antoinebhs), [SlimaneAmar](https://github.com/SlimaneAmar), [FranckLecuyer](https://github.com/FranckLecuyer)
```

Comment:
Same remark, do you need 2 new committers?

Reply:
Same answer